docs_versioned_docs/version-ros2humble/robots/accessories/sensors/cameras/axis_q6225_le.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Axis Q6225-LE
+sidebar_position: 3
+---
+
+import Support from "/components/support.mdx";
+import YAML from "/docs_versioned_docs/version-ros2humble/components/yaml/sensors/axis_camera.mdx";
+
+<center>
+  <!--<img
```

Comment:
This meant to be comment out?

Reply:
I forgot to revert that after I downloaded the appropriate image. Should be fixed now.